src/tabs/presets/DetailedDialog/PresetsDetailedDialog.js

```diff
@@ -201,7 +201,7 @@ export default class PresetsDetailedDialog {
         const groupIndex = selectedOptionKey.slice(firstUnderscoreIndex + 1, lastUnderscoreIndex);
 
         const group = this._preset.options[groupIndex];
-        if (group.isExclusive) {
+        if (group?.isExclusive) {
```

Comment:
If option is outside of a group it should be handled as such. This is quick and dirty but does not solve the root cause.

Reply:
this code is happening inside of `onBeforeClick` for all the options. Technically can catch that there is no group a couple of lines earlier in the 
```
const firstUnderscoreIndex = selectedOptionKey.indexOf('_');
const lastUnderscoreIndex = selectedOptionKey.lastIndexOf('_');
const groupIndex = selectedOptionKey.slice(firstUnderscoreIndex + 1, lastUnderscoreIndex);
```
but it will be a bit more code and extra ifs.
here i think it makes sense, we just checking if group exists.